src/textual/css/_styles_builder.py

```diff
@@ -461,10 +461,25 @@ def process_layout(self, name: str, tokens: list[Token]) -> None:
     def process_color(self, name: str, tokens: list[Token]) -> None:
         """Processes a simple color declaration."""
         name = name.replace("-", "_")
-        for token in tokens:
+        token_indexes_to_skip = []
+        for i, token in enumerate(tokens):
+            if i in token_indexes_to_skip:
+                continue
             if token.name in ("color", "token"):
+                value = token.value
+                # Color names can include digits (e.g. "turquoise4"): if the next token is a number
+                # we should consider it part of the color name:
+                next_token = tokens[i + 1] if len(tokens) > (i + 1) else None
+                if (
```

Comment:
I went for a simple and naive implementation, but don't hesitate to tell me if you think of better ways to solve this :-)

Reply:
This probably is not the fix. In tokenize.py there is a regex which parses a token as a sequence of characters, not including a number.

I think what we should do is change that so that a token must start with a-zA-z but permits digits in further characters.

Reply:
That's much better indeed! I didn't know if that would break existing behaviours or not, which is why I went for a very color-specific local fix. 
But if we can accept that "tokens" can have digits after their first char without breakinf anything, then the fix is much cleaner for sure :blush: 
Fixed in [3ed17cb](https://github.com/Textualize/textual/pull/441/commits/3ed17cb406fb48d93d240f1ef7cc9af5f1b37e5e)